llvm/include/llvm/CAS/CASDB.h

```diff
@@ -141,10 +141,11 @@ class CASDB : public CASIDContext {
   virtual CASID getID(ObjectHandle Handle) const = 0;
 
   /// Get a reference to the object called \p ID.
-  ///
-  /// Returns \c None if not stored in this CAS.
   virtual Optional<ObjectRef> getReference(const CASID &ID) const = 0;
```

Comment:
Is this change about allowing `ObjectRef` for non-existent values? If so the doc comments for `CASDB` and `ObjectRef` also need updates.  Maybe that should be a separate change entirely though.